devicetypes/smartthings/smartpower-outlet.src/smartpower-outlet.groovy

```diff
@@ -109,17 +111,19 @@ def parse(String description) {
 		if (cluster && cluster.clusterId == 0x0006 && cluster.command == 0x07){
 			if (cluster.data[0] == 0x00) {
 				log.debug "ON/OFF REPORTING CONFIG RESPONSE: " + cluster
-				sendEvent(name: "checkInterval", value: 60 * 12, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
+				event = createEvent(name: "checkInterval", value: 60 * 12, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
 			}
 			else {
 				log.warn "ON/OFF REPORTING CONFIG FAILED- error code:${cluster.data[0]}"
+				event = null
```

Comment:
This is a message we don't need/or care to propagate up to the cloud so we return null instead.